uber/models.py

```diff
@@ -890,7 +1021,7 @@ def model_mixin(cls, model):
         for name in dir(model):
             if not name.startswith('_'):
                 attr = getattr(model, name)
-                if hasattr('target', '__table__') and name in target.__table__.c:
+                if hasattr(target, '__table__') and name in target.__table__.c:
```

Comment:
This line was clearly incorrect. `hasattr('target', '__table__')` will ALWAYS return False. I have no idea what impact this will have on the rest of the system.